### PR TITLE
[DOCU-3172] MinK: Overview + UI screenshot

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

What did you change and why?

- Added content to the MinK overview page
- I'll also add a screenshot at some point, UI isn't quite there yet and doesn't have a robust example yet

 
Include any supporting resources, e.g. link to a Jira ticket, GH issue, FTI, Slack, Aha, etc.
DOCU-3172

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/konnect/mesh-manager/service-mesh.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+1. Create the global control plane in {{site.konnect_short_name}}.
+1. Add and configure a zone for your control plane.
+1. Configure `kumactl` to connect to your global control plane.
+1. Add services to your service mesh.
+
+### Prerequisites
+
+* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
+* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [Download the latest version of {{site.mesh_product_name}}](/mesh/latest/production/install-kumactl/)
```
Comment (cloudjumpercat) [inline app/konnect/mesh-manager/service-mesh.md:17]:
```suggestion
* [The latest version of {{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)
```

File: app/konnect/mesh-manager/service-mesh.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+1. Configure `kumactl` to connect to your global control plane.
+1. Add services to your service mesh.
+
+### Prerequisites
+
+* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
+* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
+<<<<<<< HEAD
+* The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)
+=======
```
Comment (cloudjumpercat) [inline app/konnect/mesh-manager/service-mesh.md:19]:
```suggestion
* The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)
```

File: app/konnect/mesh-manager/service-mesh.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+1. Add and configure a zone for your control plane.
+1. Configure `kumactl` to connect to your global control plane.
+1. Add services to your service mesh.
+
+### Prerequisites
+
+* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
+* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)
+* [Download the latest version of {{site.mesh_product_name}}](/mesh/latest/production/install-kumactl/)
```
Comment (cloudjumpercat) [inline app/konnect/mesh-manager/service-mesh.md:18]:
```suggestion
```

File: app/konnect/mesh-manager/service-mesh.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+### Prerequisites
+
+* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
+* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)
+* Mutual TLS only: If you plan to automatically encrypt [mTLS](/mesh/latest/policies/mutual-tls/) traffic for all the services in your service mesh, you will need the mTLS certificate name and authority.
+* Logging only: If you plan to add a logging backend to log traffic via the TrafficLog policy, you will need the IP address and format (WHAT IS THIS?) of your logging backend. [WHICH BACKENDS ARE AVAILABLE?]
+* Tracing only: If you plan to add a tracing backend to ___ using the TrafficTrace policy, you will need the sampling [WHAT IS THIS?] and the url of your backend. [WHICH BACKENDS ARE AVAILABLE?]
+* Metrics only: If you want to port metrics from your data plane, you will need the data plane port and path of your metrics backend. [WHICH BACKENDS ARE AVAILABLE?]
+>>>>>>> f4996ec469bff82b70f8e9b2a93f0b15c12f6c7b
```
Comment (cloudjumpercat) [inline app/konnect/mesh-manager/service-mesh.md:22]:
```suggestion
```

File: app/konnect/mesh-manager/index.md [hunk 5]
```diff
@@ -1,4 +1,31 @@
 ---
 title: About Mesh Manager
 content_type: explanation
----
\ No newline at end of file
+---
+
+Mesh Manager in {{site.konnect_product_name}} allows you to create and view your {{site.mesh_product_name}} [service meshes](/mesh/latest/introduction/what-is-a-service-mesh/) in the {{site.konnect_short_name}} UI. This allows you to use {{site.konnect_short_name}}'s federated access for platform tenants, deep API analytics, API productization with Dev Portal, and a service catalogue offering for service mesh-managed services.
```
Comment (johnharris85) [inline app/konnect/mesh-manager/index.md:6]:
```suggestion
Mesh Manager in {{site.konnect_product_name}} allows you to create, manage and view your {{site.mesh_product_name}} [service meshes](/mesh/latest/introduction/what-is-a-service-mesh/) in the {{site.konnect_short_name}} UI and using the {{site.konnect_short_name}} platform.
```

Comment (johnharris85) [reply app/konnect/mesh-manager/index.md:6]: Those capabilities are coming but don't exist today / GA.

File: app/konnect/mesh-manager/index.md [hunk 6]
```diff
@@ -1,4 +1,31 @@
 ---
 title: About Mesh Manager
 content_type: explanation
----
\ No newline at end of file
+---
+
+Mesh Manager in {{site.konnect_product_name}} allows you to create and view your {{site.mesh_product_name}} [service meshes](/mesh/latest/introduction/what-is-a-service-mesh/) in the {{site.konnect_short_name}} UI. This allows you to use {{site.konnect_short_name}}'s federated access for platform tenants, deep API analytics, API productization with Dev Portal, and a service catalogue offering for service mesh-managed services.  
+
+Mesh Manager is ideal for organizations who want to have a global control plane that allows you to run your service mesh in multiple zones. You can run a mix of Kubernetes and Universal zones. Your mesh environment can include multiple isolated service meshes (multi-tenancy) and workloads running in different regions, on different clouds, or in different datacenters.
```
Comment (johnharris85) [inline app/konnect/mesh-manager/index.md:8]:
```suggestion
Mesh Manager is ideal for organizations who want to have one or more global control planes that allow you to run your service mesh across multiple zones. You can run a mix of Kubernetes and Universal zones. Your mesh deployment environments can include multiple isolated service meshes (for multi-tenancy) and workloads running in different regions, on different clouds, or in different data-centers.
```

File: app/konnect/mesh-manager/index.md [hunk 7]
```diff
@@ -1,4 +1,31 @@
...
+After your service mesh is deployed in {{site.mesh_product_name}}, the following information will be displayed in Mesh Manager for each control plane:
+
+* Meshes and data plane proxies with [mTLS](/mesh/latest/policies/mutual-tls/)
+* RBAC
+* Zone control planes
+* [Zone Ingresses](/mesh/latest/explore/zone-ingress/)
+* [Zone Egresses](/mesh/latest/explore/zoneegress/)
+* Services associated with your service mesh
+* [Gateways](/mesh/latest/explore/gateway/) associated with your service mesh
+* Data plane proxies
```
Comment (johnharris85) [inline app/konnect/mesh-manager/index.md:27]:
```suggestion
```

Comment (johnharris85) [reply app/konnect/mesh-manager/index.md:27]: DPPs already called out on the first line

File: app/konnect/mesh-manager/service-mesh.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
+---
+title: Create a service mesh
+content_type: how-to
+---
+
+Creating a fully-functioning {{site.mesh_product_name}} global control plane in {{site.konnect_short_name}} involves the following steps:
```
Comment (johnharris85) [inline app/konnect/mesh-manager/service-mesh.md:6]:
```suggestion
Creating a fully-functioning {{site.mesh_product_name}} deployment in {{site.konnect_short_name}} involves the following steps:
```

**Review by johnharris85 — CHANGES_REQUESTED** [review]

**Review by acgoldsmith — APPROVED** [review]
LGTM - approving for merge into feature branch

**Comment by acgoldsmith** [comment]
adding back DNM as we still need the final Mesh Manager icon

File: app/konnect/mesh-manager/service-mesh.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+### Prerequisites
+
+* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
+* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* The latest version of [{{site.mesh_product_name}} installed](/mesh/latest/production/install-kumactl/)
+
+### Create and configure the global control plane
+
+1. From the left navigation menu in {{site.konnect_short_name}}, open [MESH MANAGER ICON HERE] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager).
```
Comment (acgoldsmith) [inline app/konnect/mesh-manager/service-mesh.md:21]:
```suggestion
1. From the left navigation menu in {{site.konnect_short_name}}, open [/assets/images/icons/konnect/icn-mesh-manager.svg] [**Mesh Manager**](https://cloud.konghq.com/mesh-manager).
```

**Pull request merged**